### PR TITLE
Upgrade to PF version 5

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
## Description
Upgrades version of Patternfly to version 5 releases.

## Type of change

- [x] Feature
- [x] Build related changes

### CODE REVIEW

**Comment by jeff-phillips-18** [comment]
Known issues:

- Commits list on application overview has disappearing rows.
- Unit test failures

File: package-lock.json
Comment (abhinandan13jan) [inline package-lock.json:1]: should this file be removed from PR?

Comment (jeff-phillips-18) [reply package-lock.json:1]: I don't think so. I believe we need it updated for the new versions.

**Comment by jeff-phillips-18** [comment]
@abhinandan13jan I'm not seeing that padding

**Comment by abhinandan13jan** [comment]
> @abhinandan13jan I'm not seeing that padding

Ok, it went away after a few refreshes
Screenshot 2023-08-23 at 8 44 17 PM

**Review by rottencandy — COMMENTED** [review]
Noticed some issues in add page:
- Runtime selector can't be closed
- Leave and cancel buttons aren't working, I think the event needs to be passed to onClose handlers in the modals
- Env var fields look off, maybe a browser cache issue?

**Comment by jeff-phillips-18** [comment]
@rottencandy Fixed mentioned issues. PTAL.

File: src/components/modal/DeleteResourceModal.tsx [hunk 1]
```diff
@@ -73,11 +73,9 @@ export const DeleteResourceModal: React.FC<DeleteResourceModalProps> = ({
       }) => {
         const input = values.resourceName;
         const isValid = input === resourceName;
-        const helpText = (
-          <FormHelperText className="pf-m-warning" isHidden={!(touched && !input)}>
-            {obj.kind} name missing
-          </FormHelperText>
-        );
+        const helpText = !(touched && !input) ? (
```
Comment (rottencandy) [inline src/components/modal/DeleteResourceModal.tsx:76]: Should be shown only when field is empty I think
```suggestion
        const helpText = (touched && !input) ? (
```

**Comment by jeff-phillips-18** [comment]
@rohitkrai03 I don't believe the `Learning resources` changes have anything to do with our changes.

**Review by rottencandy — APPROVED** [review]
/lgtm

**Review by rohitkrai03 — APPROVED** [review]
/approve

**Pull request merged**